src/Core_Sideloader/Core.Sideloader.cs

```diff
@@ -192,6 +202,8 @@ private void LoadModsFromDirectories(params string[] modDirectories)
 #if AI || HS2
                     UniversalAutoResolver.GenerateHeadPresetInfo(manifest, _gatheredHeadPresetInfos);
                     UniversalAutoResolver.GenerateFaceSkinInfo(manifest, _gatheredFaceSkinInfos);
+#if AI
```

Comment:
Missing code?

Reply:
Probably left over while moving code from the common codebase. As I stated, the code cleanup will be done soon since the most of features are working.

Reply:
https://github.com/IllusionMods/BepisPlugins/pull/122/commits/a57e34b7c5f0c984a134ea1e0023d8c6bfee7d86